cmd/packet_capture.go

```diff
@@ -94,12 +94,28 @@ func runPacketCaptureOnAddr(port int, filename string) {
 		if stopReceived {
 			return
 		}
+
 		go managePacketsDisplay(PcapResult{Name: filename, ByteCount: int64(len(fp.Pcap.Value)), PacketCount: 1})
 		// append new line between each record to read file easilly
-		_, err = f.Write(fp.Pcap.Value)
+		bytes, err := f.Write(fp.Pcap.Value)
 		if err != nil {
 			log.Fatal(err)
 		}
+
+		// terminate capture if max bytes reached
+		totalBytes = totalBytes + int64(bytes)
+		if totalBytes > maxBytes {
```

Comment:
do u need to handle cases where user pass only either `maxBytes` or `maxTime` but not both ?

Reply:
I was more of thinking I wanted to skip one and use the other for example if I know I am sampling large packets so I wanted to skip `maxBytes` check and just use `maxTime`?

Reply:
hm that's right ... maybe I should allow `-1` value as no limit ?

It would feel strange that the behavior of one of these two changes according to the other so I feel it's the best option here. WDYT ?